.circleci/config.yml

```diff
@@ -344,21 +344,21 @@ workflows:
           name: helm_lint
           env: 'staging'
       # Docker builds
-      - hmpps/build_docker:
+      - build-docker-image:
           <<: *only_master
           name: build-docker-image-master
+          staging: 'true'
       - build-docker-image:
           <<: *only_deploy_tags
           name: build-docker-image-release
           uat: 'true'
           preprod: 'true'
           production: 'true'
       # Deployment
-      - hmpps/deploy_env:
+      - deploy:
           <<: *only_master
           name: deploy-staging
           env: 'staging'
```

Comment:
I think this (`env: 'staging'`) was also added as part of the change to `hmpps/deploy_env` right? But it shouldn't effect switching back to `deploy`, right?

Reply:
it looks to have been there since before, this is the diff of the first commit I did on this work https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/44dfb92fe482565f84ded4eb3d1f8334c06542e7#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47

Reply:
Ahhh. Yep, I think I confused it with this change: https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/commit/0e39af15191b9d049e830061ecf631abb47c4bfe#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47